rust/osm2lanes/src/lib.rs

```diff
@@ -261,11 +269,49 @@ mod tests {
         assert!(ok);
     }
 
-    fn stringify_lane_types(lanes: &[LaneSpec]) -> String {
-        lanes.iter().map(|s| s.lane_type.as_ascii()).collect()
-    }
+    #[test]
+    fn test_roundtrip() {
+        // TODO This is brittle, depends on running the test from the right directory. Use
```

Comment:
You can remove this and the copy -- I can't find any mention of how it works, but the test runner seems to get this right and set the working directory consistently, no matter where you run `cargo test`. https://stackoverflow.com/questions/30003921/how-can-i-locate-resources-for-testing-with-cargo asked this a while ago, I'm guessing something changed since then